src/Command/CreateUserCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\User;
+use App\Repository\GroupRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Random\RandomException;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+
+#[AsCommand(
+    name: 'app:user:create ',
+    description: 'Create a user',
+)]
+class CreateUserCommand extends Command
+{
+    public function __construct(
+        private readonly EntityManagerInterface $entityManager,
+        private readonly UserPasswordHasherInterface $passwordHasher,
+        private readonly GroupRepository $groupRepository,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('username', InputArgument::REQUIRED, 'The username of the user')
+            ->addArgument('email', InputArgument::REQUIRED, 'The email of the user')
+            ->addArgument('password', InputArgument::OPTIONAL, 'The password of the user, if not set, a random password will be generated')
+            ->addOption('super', null, InputOption::VALUE_NONE, 'Add "super admin" user role to the user')
+            ->addOption('group-id', null, InputOption::VALUE_REQUIRED, 'The ID of the group to add the user to')
```

Comment:
Do you intend on adding a command listing groups and their id?

Reply:
Nope, don't think the there is time for that right now